module/skill/KickScript/src/KickScript.cpp

```diff
@@ -21,7 +21,7 @@ namespace module::skill {
     KickScript::KickScript(std::unique_ptr<NUClear::Environment> environment)
         : BehaviourReactor(std::move(environment)) {
 
-        on<Configuration>("KickScript.yaml").then([this](const Configuration& config) {
+        on<Configuration>("kickscript.yaml").then([this](const Configuration& config) {
```

Comment:
Why has this been made lowercase?

Reply:
because there is already a KickScript elsewhere

Reply:
I think rename this module to `ScriptKick`, since it's implementing a scripted kick, not a kick script.

Reply:
I'll do this in another PR then, not here.